docs/source-app/workflows/mount_cloud_object_store.rst

```diff
@@ -0,0 +1,168 @@
+    :emphasize-lines: 9-12
+
+    import lightning as L
+    from lightning_app import CloudCompute
+    from lightning_app.storage import Mount
+
+    class Flow(L.LightningFlow):
+       def __init__(self):
+           super().__init__()
+           self.my_work = MyWorkClass(
+               cloud_compute=CloudCompute(
+                   mounts=Mount(
+                       source="s3://ryft-public-sample-data/esRedditJson/",
+                       mount_path="/content/esRedditJson/",
+                   ),
+               )
+           )
+
+       def run(self):
+           self.my_work.run()
+
+You can also pass multiple mounts to a single work by passing a ``List[Mount(...), ...]`` to the
+``CloudCompute(mounts=...)`` argument.
+
+.. note::
+
+    * We enforce a hard limit of ``1,000,000`` objects which we can mount from a particular bucket prefix.
+    * The maximum size of an object in the bucket can be no more than ``5 GiB``.
+    * If multiple mounts are configured for a single ``LightningWork``, then they must each specify unique
+      ``mount_path`` arguments (a unique mount point).
+
+==============================
+Accessing Files From The Mount
+==============================
+
+Once a ``Mount`` object is passed to ``CloudCompute``, you can access, list, or read any file from the mount
+under the specified ``mount_path``, just like you would if it was on your local machine.
+
+Assuming you mount_path is ``"/content/esRedditJson/"`` you can do the following:
+
+-----------
+List Files:
+-----------
+
+.. code:: python
+
+    files = os.listdir("/content/esRedditJson/")
+
+-----------
+Read Files:
+-----------
+
+.. code:: python
+
+    with open("/content/esRedditJson/esRedditJson1", "r") as f:
+        some_data = f.read()
+
+    # do something with "some_data"...
+
+---------------------
+See the Full Example:
+---------------------
+
+.. code:: python
+    :emphasize-lines: 9-14
+
+    import os
+
+    import lightning as L
+    from lightning_app import CloudCompute
+    from lightning_app.storage import Mount
+
+    class MyWorkClass(L.LightningWork):
+       def run(self):
+           files = os.listdir("/content/esRedditJson/")
+           for file in files:
+               print(file)
+
+           with open("/content/esRedditJson/esRedditJson1", "r") as f:
+               some_data = f.read()
+               # do something with "some_data"...
+
+    class Flow(L.LightningFlow):
+       def __init__(self):
+           super().__init__()
+           self.my_work = MyWorkClass(
+               cloud_compute=CloudCompute(
+                   mounts=Mount(
+                       source="s3://ryft-public-sample-data/esRedditJson/",
+                       mount_path="/content/esRedditJson/",
+                   ),
+               )
+           )
+
+       def run(self):
+           self.my_work.run()
+
+The ``LightningWork`` component in the code above (``MyWorkClass``) would print out a list of files stored
```

Comment:
Lets just call the class "ReadMount"
you ca add this as comment to the code snppet itself so its n context